services/namex-pay/setup.cfg

```diff
@@ -75,7 +75,7 @@ max_line_length=120
 notes=FIXME,XXX,TODO
 ignored-modules=flask_sqlalchemy,sqlalchemy,SQLAlchemy,alembic,scoped_session
 ignored-classes=scoped_session
-disable=C0301,W0511,R0801,R0902
+disable=C0301,W0511,R0801,R0902,C0209
```

Comment:
I think we should not disable this check. We want to use f'{}' format in our code.